src/uct/ib/dc/verbs/dc_verbs.c

```diff
@@ -603,6 +605,19 @@ ucs_status_t uct_dc_verbs_iface_create_ah(uct_dc_iface_t *dc_iface, uint16_t lid
     return UCS_OK;
 }
 
+static UCS_F_NOINLINE void uct_dc_verbs_handle_failure(uct_ib_iface_t *ib_iface,
+                                                       void *arg)
+{
+    struct ibv_wc   *wc     = arg;
```

Comment:
no need for multiple spaces. also no need for UCS_F_NOINLINE since it's called through polymorphic function pointer anyway.